vmdk_plugin/drivers/vmdk/vmdkops/esx_vmdkcmd.go

```diff
@@ -45,6 +44,7 @@ type EsxVmdkCmd struct {
 const (
 	commBackendName string = "vsocket"
 	maxRetryCount          = 5
+	errBufLen              = 512 // The same err buf length as used in the VMCI client
```

Comment:
Actually, I've stopped using the buffer length in the client in the last changes pushed. Will remove the const used in the GO code.